test/e2e/data/infrastructure-docker/bases/cluster-with-kcp.yaml

```diff
@@ -61,7 +61,7 @@ spec:
       controllerManager:
         extraArgs: {enable-hostpath-provisioner: 'true'}
       apiServer:
-        certSANs: [localhost, 127.0.0.1, 0.0.0.0]
+        certSANs: [localhost, 127.0.0.1, 0.0.0.0, host.docker.internal]
```

Comment:
Why is this necessary now?

Reply:
This is required by the kubetest binary

Reply:
more specifically, it's required by kubetest when running on MacOS because of the way ports are proxied.

Reply:
Seems like it would be helpful to add a comment here describing why adding `host.docker.internal` is needed?

Reply:
done